### DISCUSSION TITLE
Select record from IndexedDB with q-dialog and q-card doesn't keep ID

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#11233] [General - Components / Directives / etc]
I want to display a confirm dialog so a user can delete a particular record from the database.

When I use the suggested example for the q-dialog, it loses the ID for the row I want to delete.  

I'm using a v-for to grab the values and display them in a table which works fine.
If I make just a regular button for the delete, the values are passed correctly and the selected record is deleted. 
However if I throw in a dialog and a card, the value is lost and only the last record in the database is deleted.

How do I retain the value inside the q-card?  I've included various displays so I can see what value I'm getting where.  
The "Table 1" and "Table 2" displays contain the correct key value.  But when it pops down into the q-card, "Dialog 1" and "Dialog 2" show the last value in the table, which is not what I want.  Therefore the @click event doesn't send the value I want to the method to delete the correct record.

I'd love some help thanks!

```
 <tr v-for="entryForPeople in peopleArray" :key="entryForPeople.pk_person">
<td..>

     <q-btn
          round
          size="10px"
          padding="xs"
          icon="las la-trash"
          color="red-6"
          @click="confirmPersonDelete = true" 
          / >  

          Table 1:  {{ entryForPeople.pk_person }}

         <q-dialog v-model="confirmPersonDelete" persistent>

           Table 2:  {{ entryForPeople.pk_person }}

        <q-card>
            <q-card-section class="row items-center">
            <q-avatar icon="las la-trash" color="primary" text-color="white" />
            <span class="q-ml-sm">Are you sure you want to delete this person?</span>

            Dialog 1: {{ entryForPeople.pk_person }}

            </q-card-section>
            <q-card-actions align="right">

             Dialog 2: {{ entryForPeople.pk_person }}

            <q-btn
              flat
              label="Yes Delete"
              color="primary"
              @click="deletePerson(entryForPeople.pk_person)"
              v-close-popup />                            
            <q-btn
              flat
              label="Cancel"
              color="primary"
              v-close-popup />                            
            </q-card-actions>
        </q-card>

</td>
</tr>
```

**Comment by hinogi** [comment]
Do you maybe have a runable example?
